google/cloud/spanner/options.h

```diff
@@ -232,6 +232,15 @@ struct RequestPriorityOption {
   using Type = spanner::RequestPriority;
 };
 
+/**
+ * Option for `google::cloud::Options` to set a maximum batching delay.
+ *
+ * @ingroup spanner-options
+ */
+struct MaxBatchingDelayMsOption {
+  using Type = int;
+};
+
```

Comment:
I do not think we would want to use `int` to represent time intervals:

```suggestion
struct MaxBatchingDelayOption {
  using Type = std::chrono::milliseconds;
};

```